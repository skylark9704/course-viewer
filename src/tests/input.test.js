import React from 'react';
import renderer from 'react-test-renderer'
import InputComponent from "../components/common/Input";

// Snapshot Testing
it('renders input', () => {
    const tree = renderer.create(<InputComponent name="input" />).toJSON()

    expect(tree).toMatchSnapshot()
    expect(tree.children[0].type).toBe('input')
    expect(tree.children[0].props.type).toBe('text')
})