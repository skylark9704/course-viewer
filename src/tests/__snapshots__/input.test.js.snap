// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders input 1`] = `
<div
  className="form-group"
>
  <input
    className="form-control"
    defaultValue=""
    name="input"
    type="text"
  />
</div>
`;
